templates/ui/experiment.html

```diff
@@ -37,12 +37,14 @@ <h1>Experiment <b>{{ experiment.name }}</b></h1>
                             <th>Mode:</th>
                             <td>{{ experiment.mode }}</td>
                         </tr>
-                        {% if experiment.assigned_to %}
```

Comment:
Why was this `if` removed?

Reply:
Good catch.

An earlier version only showed the experiment's requirements when it was unassigned. When I removed that logic, I took out the whole conditional by mistake.

Reply:
Do you want a dash as a placeholder here as well? Currently it's just a blank line if the experiment has no requirements.

Reply:
Either add a dash or just hide the row.